api/v1alpha1/imageupdateautomation_types.go

```diff
@@ -92,6 +94,21 @@ type ImageUpdateAutomationStatus struct {
 	// made).
 	// +optional
 	LastAutomationRunTime *metav1.Time `json:"lastAutomationRunTime,omitempty"`
+	// +optional
+	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
+	// +optional
+	Conditions []metav1.Condition `json:"conditions,omitempty"`
+}
+
+const (
```

Comment:
In the other component controllers, these are defined in a separate `condition_types.go` file, I suppose it's not necessarily an officially accepted convention but rather just a relic from the past.

It would be good to have these documented, a short description of what they represent.

Reply:
I gave each a golint-compatible comment.